tests/test_pool.py

```diff
@@ -15,10 +16,17 @@ class FakeXYZBackend(ThreadSafeCreateMixin):
     id = 'fake_xyz'
 
 
+def configure(pools):
+    if settings:
+        settings.POOL_OF_RAMOS = pools
+    else:
+        _configure(pools=pools)
+
+
 class TestBackendPool(object):
 
     def test_get_backends_should_return_all_backends_instances(self):
-        settings.POOL_OF_RAMOS = {
+        configure(pools={
```

Comment:
You've removed the use of settings in unit tests. What do you think about adding another test file? (or fixtures, if you want to reuse some test code)